filecheck/checker.ts

```diff
@@ -256,48 +243,102 @@ export async function checkFile(
   }
 }
 
-function canCheckFile(filePath: string) {
-  const filePathParts = filePath.split(path.sep);
-
+function isRelevantFile(filePath: string) {
   return (
-    filePathParts.includes("files") &&
-    !filePathParts.includes("node_modules") &&
-    !/\.(DS_Store|html|json|md|txt|yml)$/i.test(filePath)
+    (filePath.includes(`files${path.sep}en-us`) ||
+      filePath.includes(`files${path.sep}jsondata`)) &&
+    !/\.(DS_Store|ini)$/i.test(filePath)
   );
 }
 
+function isBinaryFile(filePath: string) {
+  return !/\.(DS_Store|html|json|md|txt|yml)$/i.test(filePath);
+}
+
 async function resolveDirectory(file: string): Promise<string[]> {
   const stats = await fs.lstat(file);
   if (stats.isDirectory()) {
     const api = new fdir()
       .withErrors()
       .withFullPaths()
-      .filter((filePath) => canCheckFile(filePath))
+      .filter((filePath) => isRelevantFile(filePath))
       .crawl(file);
     return api.withPromise() as Promise<PathsOutput>;
-  } else if (stats.isFile() && canCheckFile(file)) {
+  } else if (stats.isFile() && isRelevantFile(file)) {
     return [file];
   } else {
     return [];
   }
 }
 
+function validatePath(filePath: string): string[] {
+  const reasons = [];
```

Comment:
Let's rename the variable to `errors`, since that's what we're essentially collecting here:

```suggestion
  const errors = [];
```